src/Equinor.ProCoSys.IPO.Infrastructure/Repositories/InvitationRepository.cs

```diff
@@ -39,6 +40,11 @@ public void MoveCommPkg(string fromProject, string toProject, string commPkgNo,
                     .Where(i => i.ProjectName == fromProject &&
                                 (i.CommPkgs.Any(c => c.CommPkgNo == commPkgNo) || i.McPkgs.Any(m => m.CommPkgNo == commPkgNo))).ToList();
 
+            if (invitationsToMove.Any(i => i.CommPkgs.Count()>1) || invitationsToMove.Any(i => i.McPkgs.Any(m => m.CommPkgNo!=commPkgNo)))
```

Comment:
what are you checking after or in this statement? finding it hard to read/understand